apps/testclient/templates/access-denied.html

```diff
@@ -0,0 +1,44 @@
+{% load i18n %}
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>{{ settings.APPLICATION_TITLE }} Test Client Error - Missing Token</title>
+    <meta name="author" content="Alan Viars">
+    <!-- FavIcons generated by http://realfavicongenerator.net -->
+   <link href="{% static 'img/favicon/favicon.ico' %}" rel="shortcut icon" type="image/x-icon" />
+   <link rel="apple-touch-icon" sizes="180x180" href="{% static 'img/favicon/apple-touch-icon.png' %}">
+   <link rel="icon" type="image/png" href="{% static 'img/favicon/favicon-32x32.png" sizes="32x32' %}">
+   <link rel="icon" type="image/png" href="{% static 'img/favicon/favicon-16x16.png" sizes="16x16' %}">
+   <link rel="manifest" href="{% static 'img/favicon/manifest.json' %}">
+   <link rel="mask-icon" href="{% static 'img/favicon/safari-pinned-tab.svg' %}" color="#155eeb">
+   <link rel="shortcut icon" href="{% static 'img/favicon/favicon.ico' %}">
+   <meta name="msapplication-config" content="{% static 'img/favicon/browserconfig.xml' %}">
+   <link rel="stylesheet" href="{% static 'design-system/packages/core/dist/index.css' %}" />
+   <link rel="stylesheet" href="{% static 'design-system/style.css' %}">
+   <style media="screen" type="text/css">
+		    body{
+						margin-left: 20px;
+					 margin-top: 40px;
+						}
+		 </style>
+</head>
+
+
+
+<body class="ds-base">
+ <h1>Missing Token</h1>
```

Comment:
Can you just say, "Access denied?" and then say, "The user denied access the this application." This page is developer facing and only serves one purpose, so we might as well be explicit.

Reply:
oops I had it that way but I hosed it with all the git stuff.
will patch

Reply:
Fixed.